docs/developer/quick-start.md

```diff
@@ -198,6 +113,3 @@ The `handle` code in this example
 [command]: commands.md (Atomist Command Automations)
 [event]: events.md (Atomist Event Automations)
 [graphql-api]: graphql.md (Atomist Automation GraphQL)
-
```

Comment:
Is the link for `[slack]` somewhere else in the file?

Reply:
yes